ui/packages/consul-ui/app/components/consul/kv/form/index.hbs

```diff
@@ -19,11 +20,11 @@
 {{#if api.isCreate}}
         <label class="type-text{{if api.data.error.Key ' has-error'}}">
             <span>Key or folder</span>
-            <input autofocus="autofocus" type="text" value={{left-trim api.data.Key parent.Key}} name="additional" oninput={{action api.change}} placeholder="Key or folder" />
+            <input autofocus="autofocus" type="text" value={{left-trim api.data.Key parent}} name="additional" oninput={{action api.change}} placeholder="Key or folder" />
```

Comment:
I think I changed this so that `parent` is the same as `parent.Key`. We only ever used `parent.Key` so figured it was easier to just use `parent` for a name.